Sources/SemanticAnalyzer/SemanticAnalyzer+Expressions.swift

```diff
@@ -63,6 +63,7 @@ extension SemanticAnalyzer {
 
   public func process(functionCall: FunctionCall, passContext: ASTPassContext) -> ASTPassResult<FunctionCall> {
     let environment = passContext.environment!
+    let enclosingType = passContext.enclosingTypeIdentifier!.name
```

Comment:
Do we always have a named enclosing type identifier? I can understand why environment might be forcibly unwrapped but if you're forcibly unwrapping this I think a comment to explain why you think it won't crash would be helpful.